static/app/views/issueDetails/utils.tsx

```diff
@@ -51,26 +51,40 @@ export function useDefaultIssueEvent() {
 /**
  * Combines two TagValue arrays and combines TagValue.count upon conflict
  */
-export function mergeTagValues(
+export function mergeAndSortTagValues(
   tagValues1: TagValue[],
-  tagValues2: TagValue[]
+  tagValues2: TagValue[],
+  sort: 'count' | 'lastSeen' = 'lastSeen'
 ): TagValue[] {
   const tagValueCollection = tagValues1.reduce<Record<string, TagValue>>(
     (acc, tagValue) => {
-      acc[tagValue.name] = tagValue;
+      acc[tagValue.value] = tagValue;
       return acc;
     },
     {}
   );
   tagValues2.forEach(tagValue => {
-    if (tagValueCollection[tagValue.name]) {
-      tagValueCollection[tagValue.name].count += tagValue.count;
+    if (tagValueCollection[tagValue.value]) {
+      tagValueCollection[tagValue.value].count += tagValue.count;
+      if (
+        new Date(tagValue.lastSeen) >
```

Comment:
No need to use the Date class here, ISO format is sortable alphabetically, so you can just compare the string values (very convenient!)